.github/workflows/precious.yml

```diff
@@ -0,0 +1,45 @@
+name: Precious
```

Comment:
It wasn't clear to me why the `linting.yml` file didn't fail with all of the errors that were fixed in this PR though.

Reply:
> It wasn't clear to me why the linting.yml file didn't fail with all of the errors that were fixed in this PR though.

It's because warnings don't return a non-zero exit code.

To fix this in stylelint, change [defaultSeverity](https://stylelint.io/user-guide/configure/#defaultseverity) to `error`.

For eslint, add the `--max-warnings=0` cli argument

Reply:
Adding something for typos does seem ideal for something like the blog site.

Thanks, I've removed the precious github action in https://github.com/maxmind/blog-site/pull/915/commits/b164d44d2b30739590ed6ab66c04eab2bec73f17 if it is preferred to use only the linting github action.